docs/conf.py

```diff
@@ -68,11 +79,7 @@
 copyright = '2016, Giulio Ungaretti, Alex Johnson'
 author = 'Giulio Ungaretti, Alex Johnson'
```

Comment:
the founding fathers :)

Reply:
@QCoDeS/core Yes, should it include all core devs or should we simply change it to `the QCoDeS developers`?

Reply:
I have no opinion about this :)

but it's probably fair to say that nobody is holding the copyright.

Reply:
Yes I think copyright should just be QCoDeS developers of some such generic thing